Utils/download_packs_and_docker_images.py

```diff
@@ -146,7 +146,9 @@ def options_handler():
     """
     parser = argparse.ArgumentParser(description="Downloads XSOAR packs as zip and their latest docker images as tar.")
     parser.add_argument('-p', '--packs',
-                        help="Comma separated list of pack names as they appear in https://xsoar.pan.dev/marketplace",
+                        help="A list of pack names as they appear in https://xsoar.pan.dev/marketplaceEither provided "
+                             "via a path to a file that contains the packs list (separated by new lines) or "
+                             "a string of comma separated packs (e.g. Base,AutoFocus)",
```

Comment:
@ShirleyDenkberg Please take a look at this recent change to the script in regards to https://github.com/demisto/content-docs/pull/920. This should be the last change before merging.